packages/core/spec/screenplay/interactions/Wait.spec.ts

```diff
@@ -1,6 +1,8 @@
+/* eslint-disable unicorn/consistent-function-scoping */
```

Comment:
TS compiler in VS Code complains about the arrow functions are not in scope without this and you have it already in other specs https://github.com/serenity-js/serenity-js/blob/e61091eca7bdd3d93dedf14fd26eb3a1f58a3435/integration/web-specs/spec/screenplay/models/PageElements.patterns.spec.ts#L1

Reply:
Ah yes, so I'm in two minds about this ESLint recommendation. It's fair for "production code", but I feel like in tests it's easier to have helper functions close to the test they're helping rather than have them in the outside scope only for the sake of pleasing ESLint. Disabling this rule on case by case basis in tests is fine 👍🏻